fiberzap/go.mod

```diff
@@ -3,21 +3,24 @@ module github.com/gofiber/contrib/fiberzap
 go 1.18
 
 require (
-	github.com/gofiber/fiber/v2 v2.47.0
+	github.com/gofiber/fiber/v2 v2.47.1-0.20230626061657-fefc5338347d
```

Comment:
lets wait for the next release

Reply:
ok

Reply:
new version is v2.48.0
https://github.com/gofiber/fiber/releases/tag/v2.48.0